addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -66,14 +71,17 @@
 
     private InetAddress ifAddress;
     private CommandProcessor commandProcessor;
-    ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_DEVICES);
-    ScheduledFuture<?> scheduledFuture;
+    private ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_THREADS);
```

Comment:
I see that you didn't introduce it in this PR, but in general, I would advice to use `ThreadPoolManager.getScheduledPool` as this also nicely names the threads and allows people to reconfigure the pool size (default is 5, which should work here as well, right?).